site/docs/pages/wallet/types.mdx

```diff
@@ -82,6 +82,21 @@ export type WalletDropdownDisconnectReact = {
 };
 ```
 
+## `WalletDropdownFundLinkReact`
+
+```ts
+export type WalletDropdownFundLinkReact = {
+  className?: string; // Optional className override for the element
+  href: string;
+  icon?: ReactNode;
+  rel?: string;
+  openIn?: 'popup' | 'tab'; // Whether to open the funding flow in a tab or a popup window
+  target?: string; // Where to open the target if `openIn` is set to tab
+  text?: string; // Optional text override
+  popupSize?: 'sm' | 'md' | 'lg'; // Size of the popup window if `openIn` is set to `popup`
```

Comment:
I'm wondering if we should even allow the popupSize to be adjustable. 

It seems to add a bit of complexity. 

What if we only supported the "recommended" size - what we think looks best?

Reply:
not opposed to just hardcoding since we already allow the user to override the `windowFeatures` parameter on `window.open` using the `popupFeatures` prop

Reply:
but IMO having to perform this calculation as a developer is pretty annoying, we should have some sort of pre-built responsive design BUT still allow the user full customizability

Reply:
+ to having a pre-build responsive design

What type of size customizability would the user want? 

Personally, as long as the popup window looks "normal" and everything is in view then I would be happy with it.

Reply:
How the popup looks like? I didn't see a screenshot in the recap. Can we see how each form looks like.

Reply:
added screenshots on desktop